### PR TITLE
Moremeats

### PR DESCRIPTION
**Pull request by FelixVita**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # shoppinglist
 
 salmon
-beef
\ No newline at end of file
+beef
```
Comment (FelixVita) [inline README.md:5]: can we have chicken instead of beef?

Comment (FelixVita) [reply README.md:5]: never mind

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+veal
```
Comment (ogeretla) [inline README.md]: I'm not cool with baby meats. Change this to some other meat please.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # shoppinglist
 
 salmon
```
Comment (ogeretla) [inline README.md]: I hate fish. Please change.

**Review by ogeretla — CHANGES_REQUESTED** [review]
You need to change a few things Felix.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
\ No newline at end of file
+turkey
+
+beef
+
+lamb
+
+tomatoes
+
+onions
```
Comment (ogeretla) [inline README.md]:
```suggestion
red onions
```
Let's be specific

Comment (FelixVita) [reply README.md]: good suggestion

**Review by ogeretla — APPROVED** [review]

**Comment by FelixVita** [comment]
awesome sauce!

**Pull request merged**